pallas-codec/src/utils.rs

```diff
@@ -1340,6 +1341,12 @@ impl From<Bytes> for Vec<u8> {
     }
 }
 
+impl<const N: usize> From<&Bytes> for [u8; N] {
+    fn from(b: &Bytes) -> Self {
+        b.0.as_slice()[..N].try_into().expect("Infallible")
+    }
+}
```

Comment:
I don't think we can consider this conversion infallible. Wouldn't this panic if bytes len is less that N?

Reply:
Yeah. I'll convert it to TryFrom.

Reply:
fixed